store/postgres/tests/store.rs

```diff
@@ -2237,3 +2256,123 @@ fn cleanup_cached_blocks() {
         Ok(())
     })
 }
+
+#[test]
+fn reorg_tracking() {
+    fn update_john(store: &Arc<DieselStore>, age: i32, block: &EthereumBlockPointer) {
+        let test_entity_1 = create_test_entity(
+            "1",
+            USER,
+            "Johnton",
+            "tonofjohn@email.com",
+            age,
+            184.4,
+            false,
+            None,
+        );
+        transact_entity_operations(
+            store,
+            TEST_SUBGRAPH_ID.clone(),
+            block.clone(),
+            vec![test_entity_1],
+        )
+        .unwrap();
+    }
+
+    macro_rules! check_state {
```

Comment:
Why can't this macro be a function?

Reply:
It's a macro so that you get accurate line numbers on test failure, i.e., the line number with the `check_state!` rather than the line number of the failed assert inside `fn check_state`